packages/router-generator/tests/generator/custom-tokens/routeTree.snapshot.ts

```diff
@@ -127,15 +127,93 @@ declare module '@tanstack/react-router' {
 
 // Create and export the route tree
 
-export const routeTree = rootRoute.addChildren({
-  R1nd3xRoute,
-  BlogR0ut3Route: BlogR0ut3Route.addChildren({ BlogSlugRoute, Blog1nd3xRoute }),
-  PostsR0ut3Route: PostsR0ut3Route.addChildren({
-    Posts1nd3xRoute,
-    PostsPostIdDeepRoute,
-    PostsPostId1nd3xRoute,
-  }),
-})
+const R1nd3xRouteWithChildren = R1nd3xRoute
```

Comment:
should we maybe not generate that line for leave routes?

Reply:
Yeah, I'll remove these from the generator